hazelcast-sql/src/test/java/com/hazelcast/sql/misc/SqlNoDeserializationTest.java

```diff
@@ -124,9 +113,10 @@ private static PortableFactory portableFactory() {
         };
     }
 
+    @Ignore(value = "Doesn't work as expected for Jet engine.")
```

Comment:
Forgotten `@Ignore`?

Reply:
Nope, it has following issue : https://github.com/hazelcast/hazelcast/issues/19273
Me or someone else will do this optimization in a separate PR targeted for 5.1.